WTRSimlib/src/main/java/simulation/launch/SocketHandler.java

```diff
@@ -0,0 +1,119 @@
+package simulation.launch;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import simulation.launch.UnityVerifier;
+
+// Credit to the University of Northampton, UK
+// http://www.eng.northampton.ac.uk/~espen/CSY2026/JavaServerCSClient.htm
+// CSY2026 Modern Networks
+
+/**
+ * Class that handles communication with Unity.
+ */
+class SocketHandler implements Runnable {
```

Comment:
any particular reason it's package-private?

Reply:
No.

Reply:
For the sake of consistency, I'm making classes that don't need to be used outside of the launch package, package private